### ISSUE TITLE
passingRatePerLaunch widget counts total and do not use contentFields

### ISSUE THREAD

**Issue by fenchu** [reportportal/reportportal#1832]
`If` you use the passingRatePerLaunch it counts passed per total, it should count passed as of passed+failed,
it makes no point in counting skipped cases, the statisticTrend has this, but not passingRatePerLaunch.

I saw the contentField was empty so I tried setting is as in statisticTrend widget. but it did not help.
I want to have a wall of tests and only if there are any red I will go into them.

clicking and getting to the launcher like in statisticTrend do not work either.

```
    "passingRatePerLaunch": {
        "contentParameters": {
            "contentFields": [
                "statistics$executions$passed",
                "statistics$executions$failed"
            ],
            "itemsCount": 1,
            "widgetOptions": {
                "viewMode": "bar",
                "launchNameFilter": "##filter##"
            }
        }
    }
```

**Screenshots**

![passing-rate-per-launch](https://user-images.githubusercontent.com/11349883/187679798-2f3dc690-dc3e-4c83-9c2a-a06607618e67.jpg)

**Versions:**
 - OS, Browser - Ubuntu 20.04LTS server, any browser
 - Version of RP [find it under Login form, copy as is] - 5.7.2 docker container

**Comment by DzmitryHumianiuk** [comment]
@fenchu hey, can't get you point.

so the issue is that `Passing rate per launch` widgets shows skipped test cases as a failed(non-passed) cases.
is it right?
and you expect to see only passed and failed there. Without skipped been identified as NON-passed.

**Comment by fenchu** [comment]
correct but you cant define it. skipped tests are in many cases irrellevant, other widget you can define what statuses to include.

**Comment by DzmitryHumianiuk** [comment]
@fenchu that's right.
looks like we need a configuration for this widget.
radio button to switch between:
1. Passed and Failed
2. Passed and Non-passed (Failed + Skipped)

**Comment by VolhaKarenko** [comment]
The configuration to create the widget with the ratio based on total test cases or total test cases excluding 'Skipped' will be available in version 5.8.
